src/EditorFeatures/CSharpTest/DocumentationComments/DocumentationCommentTests.cs

```diff
@@ -2778,6 +2778,108 @@ class C
         VerifyTypingCharacter(code, expected);
     }
 
+    [WpfFact, WorkItem("https://github.com/dotnet/roslyn/issues/75838")]
+    public void TypingCharacter_ExistingText1()
+    {
+        var code =
+            """
+            /// foo$$
+            class C
+            {
+            }
+            """;
+
+        var expected =
+            """
+            /// foo/$$
+            class C
+            {
+            }
+            """;
+
+        VerifyTypingCharacter(code, expected);
+    }
+
+    [WpfFact, WorkItem("https://github.com/dotnet/roslyn/issues/75838")]
+    public void TypingCharacter_ExistingText2()
+    {
+        var code =
+            """
+            namespace N
+            {
+                /// foo$$
+                class C
+                {
+                }
+            }
+            """;
+
+        var expected =
+            """
+            namespace N
+            {
+                /// foo/$$
+                class C
+                {
+                }
+            }
+            """;
+
+        VerifyTypingCharacter(code, expected);
+    }
+
+    [WpfFact, WorkItem("https://github.com/dotnet/roslyn/issues/75838")]
+    public void PressingEnter_ExistingText1()
+    {
+        var code =
+            """
+            /// foo$$
+            class C
+            {
+            }
+            """;
+
+        var expected =
+            """
+            /// foo
+            /// $$
+            class C
+            {
+            }
+            """;
+
+        VerifyPressingEnter(code, expected);
+    }
+
+    [WpfFact, WorkItem("https://github.com/dotnet/roslyn/issues/75838")]
+    public void PressingEnter_ExistingText2()
+    {
+        var code =
+            """
+            namespace N
+            {
+                /// foo$$
+                class C
+                {
+                }
+            }
+            """;
+
+        var expected =
+            """
+            namespace N
+            {
+                /// foo
+                /// $$
```

Comment:
As I can see from the written test, the expected behaviour is going to be that it generated another broken xml-doc line instead of a proper summary-wrapped xml doc, as what triple slashes are there for?

We already have an analyzer/code-completion that turns
```cs
/// Bla
class C { }
```
into
```cs
/// <summary>
/// Bla
/// </summary>
class C { }
```
if we remove and re-add one of the slashes in the line. 

However, if we have two lines within our broken summary, like so
```cs
/// Bla
/// Bla Bla
class C { }
```
removing and re-adding one of the slashes will turn it into a half-fixed summary block
```cs
/// <summary>
/// Bla
/// </summary>
/// Bla Bla
class C { }
```

This could make sense, if the lower summary xml docs were wrapped in other / already existing xml doc tags, but they are not, thus they should always belong to the upper generated summary xml. 
Those are some of my thoughts on the behaviour, and why I dont think this fix necessarily fixes the underlying issue completely